### PR TITLE
Rock

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

Introduces the glorious rockfruit, a delish little fruit which just happens to grow literal granite around it.

image
image

I'm not the greatest spriter but I think these look good enough. It is honestly hard to work with a desaturated, rocky palette when floors use the same

## Why It's Good For The Game

More interesting food is good, plus this fits nicely into adding stone type food later (Very soon)

## Changelog

:cl:
add: Rockfruit, the new awesome fruit that's literally just a plant growing a granite shell

image: Rockfruit seeds, plant, fruit, and rocks
/:cl:

### CODE REVIEW

**Comment by BurgerLUA** [comment]
Can't wait for this to be merged so I can add the ability to grow diamonds by mutating this plant.

image

File: modular_zubbers/code/modules/hydroponics/grown/rocks.dm [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	force = 5 // Comparatively shit considering a nettle is 15
+	throwforce = 10 // Less shit but hey, it is a rock
+
+	icon = 'modular_zubbers/code/modules/hydroponics/icons/harvest.dmi'
+	icon_state = "rockfruit"
+
+	var/product = /obj/item/food/grown/rockfruit
+
+/obj/item/grown/rockfruit/attack_self(mob/user, modifiers)
+	user.show_message(span_notice("You begin peeling the rocky exterior..."))
```
Comment (Cyprex) [inline modular_zubbers/code/modules/hydroponics/grown/rocks.dm:49]:
```suggestion
	user.show_message(span_notice("You take out your pocket jackhammer and start hammering away"))
```
This is fine, I just felt like suggesting something silly like this, I suppose.

File: modular_zubbers/code/modules/vending/megaseed.dm [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/obj/machinery/vending/hydroseeds/Initialize(mapload)
+	for(var/category in product_categories)
+		if(category["name"] == "Fruits")
+			category["products"] += list(/obj/item/seeds/rockfruit = 3)
```
Comment (Cyprex) [inline modular_zubbers/code/modules/vending/megaseed.dm:4]: This feels pretty ugly but it is what it is.

**Review by Cyprex — APPROVED** [review]
Looks good to me, ready to merge!

File: modular_zubbers/code/modules/vending/megaseed.dm (hunk 2 above)
Comment (Majkl-J) [reply modular_zubbers/code/modules/vending/megaseed.dm:4]: Can't do it much better unfortunately

**Comment by projectkepler-ru** [comment]
is this some kind of reference I'm too old to get

**Comment by Majkl-J** [comment]
Also I got yelled at by an unnamed third party my sprites sucked, this will be fixed in rock part 2

**Pull request merged**